### DISCUSSION TITLE
能不能支持一下语音的情绪控制功能

### DISCUSSION THREAD

**Discussion by scientificCommunity** [svc-develop-team/so-vits-svc#88] [General]
模型训练时提取情绪属性，语音输出时能够自动根据上下文加上情绪。或者文本里通过标签指定情绪

**Comment by PriesiaMioShirakana** [comment]
请问，您是否走错了仓库。这里是SoVitsSvc，是一个Svc模型，而不是Text2Speech。
Svc推理结果的感情已经由输入音频控制了

**Comment by Alwin-Lin** [comment]
這邊: https://github.com/innnky/emotional-vits

**Reply by Miuzarte** [reply]
tts和svc不要混淆比较好
